gdbgui/static/js/gdbgui.js

```diff
@@ -2350,6 +2350,7 @@ const Expressions = {
             let children = r.payload.children.map(child_obj => Expressions.prepare_gdb_obj_for_storage(child_obj, expr_type))
             // save these children as a field to their parent
             parent_obj.children = children
+            parent_obj.numchild = children.length
```

Comment:
Nevermind, I was just looking in the local file. You are right, numchild is used in many places.